docs/advanced/index.rst

```diff
@@ -1,17 +1,18 @@
-=====================
-Getting More Involved
-=====================
+================================================
+Technical Guide to Development and Documentation
+================================================
 
 If you've made it this far without `saying hello <http://openhatch.readthedocs.org/en/latest/community/contact.html>`_
 you should definitely do so!
 
 .. toctree::
-    :maxdepth: 2
+   :maxdepth: 2
 
-    advanced_installation
-    layout_source_code
-    deployment
-    heroku
-    maintenance
-    adding_new_bug_tracker_git
-    advanced_testing
+   styleguide
```

Comment:
When I build this locally, I guet:

/home/paulproteus/projects/oh-mainline/docs/index.rst:56: WARNING: toctree contains reference to nonexisting document u'advanced/styleguide'

Is this the rST styleguide you mentioned? I think you didn't `git add` it.

FWIW, if possible, I would prefer to include that by reference, rather than including a copy, but I can live with it either way.